content/partner/aws.md

```diff
@@ -1,9 +1,112 @@
 ---
-title: Cloud Native AWS Infrastructure as Code
+title: Modern Infrastructure as Code for AWS
 layout: aws
 url: /aws
 
 meta_desc: Pulumi provides huge productivity gains and a unified programming model for Devs and DevOps, through infrastructure as code on the AWS cloud.
+
+hero:
+    title: Cloud Engineering for AWS
+    description: |
+        Pulumi's infrastructure as code SDK helps create, deploy, and manage your
+        AWS infrastructure, including containers, serverless functions, and
+        other infrastructure using modern programming languages.
+    cta_text: See what's new
+    cta_url: "/blog/announcing-nextgen-azure-provider"
```

Comment:
Why link to Azure-related post from AWS page?

Reply:
Ah right, forgot to remove the old content that is not used anymore. Good catch!